router-bridge/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "2.0.0-alpha.5",
   "description": "Apollo Router JS Bridge Entrypoint",
   "scripts": {
-    "build": "mkdir -p bundled js-dist && tsc && node esbuild/bundler.js",
```

Comment:
One of the things that `-p` offers is that it will not error when the directory already exists.  We may want that to be maintained for local development reasons?  Maybe we should just use [`make-dir-cli`](https://www.npmjs.com/package/make-dir-cli) or put [`make-dir`](https://npm.im/make-dir) into `esbuild/bundler.js`?

Reply:
Indeed! I was wondering to what extent we would want to rely on "more tools", but it definitely looks the correct way to go

> I mention this is a workaround because I feel like depending on how complex the build script becomes, we might want to leverage [cross-env](https://www.npmjs.com/package/cross-env) or other packages to run OS specific commands

Let's give this a try